rules/no-zepto-ajax.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+/**
+ * Reports the given node and identifier name.
+ * @param {RuleContext} context The ESLint rule context.
+ * @param {ASTNode} node The node to report on.
+ * @param {string} identifierName The name of the identifier.
+ * @returns {void}
+ */
+function report(context, node, identifierName) {
+    context.report(node, "Unexpected {{name}}.", { name: identifierName });
+}
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow the use of `alert`, `confirm`, and `prompt`",
+            category: "Best Practices",
```

Comment:
there seems no relationship between the description and the rule,

and change category to: "Stylistic Issues"